core/arch/aarch64/emit_utils.c

```diff
@@ -107,6 +107,8 @@ insert_exit_stub_other_flags(dcontext_t *dcontext, fragment_t *f,
     /* FIXME i#1575: coarse-grain NYI on ARM */
     ASSERT_NOT_IMPLEMENTED(!TEST(FRAG_COARSE_GRAIN, f->flags));
     if (LINKSTUB_DIRECT(l_flags)) {
+        /* We put a NOP here for future linking. */
```

Comment:
But linking was already working fine, so it is unclear why we'd need a nop now.  Please elaborate in the comment -- maybe best in the pre-function comment.  Putting an asm listing is also helpful (esp with this stub getting very large: 8 instructions now).  This is unusual.